test/minimal/test.js

```diff
@@ -0,0 +1,87 @@
+var assert = require('chai').assert;
+var testGen = require('../../index.js').testGen;
+var swagger = require('./swagger.json');
+var lint = require('mocha-eslint');
+
+describe('minimal swagger', function(){
+	describe('request-option', function(){
+		describe('pathNames-option', function(){
+			it ('should still generate all paths from empty pathNames option', function(done){
+        var output = testGen(swagger, {
+          'assertionFormat':'should',
+          'pathNames':[],
+          'testmodule':'request'
+           ,'destination':'./test/minimal'
+        });
+
+        assert.isArray(output);
```

Comment:
the describe block should say: `"testing with should and request"`:

almost each assertion should be wrapped in a `it` block.